TESTING-INSTRUCTIONS.md

```diff
@@ -2,6 +2,17 @@
 
 ## Unreleased
 
+### Display WCPay task when installed via subscriptions option on profiler
+
+1. Start with a fresh install.
+2. Navigate to WooCommerce -> Home to start the OBW.
+3. Ensure you select "subscriptions" under the step for product types.
+4. Complete the rest of the onboarding wizard.
+5. Confirm that "Get paid with WooCommerce Payments" appears under primary task list.
+6. When clicked, it should redirect to WCPay connection page.
+6. Confirm that "Set up additional payment providers" appears under extended task list.
```

Comment:
Nitpick: This task won't be visible if `WC Pay` is [not set up (connected)](https://github.com/woocommerce/woocommerce-admin/issues/8414).

Reply:
Ah, understood. It looks like this is a current problem already captured by the issue you linked, so I'll just remedy the instructions with the assumption that it's working properly. Done in https://github.com/woocommerce/woocommerce-admin/pull/8445/commits/54af50713a6ef93a244ab632fa874cacd7c72fea.